### PR TITLE
Feature/subsume base image

### PR DESCRIPTION
**Pull request by bilow-factual**
Base on phusion image rather than boritzio, pull up dependencies from @bfs docker-base image

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
+
+RUN add-apt-repository ppa:openjdk-r/ppa && \
+    apt-get update
+
 RUN update-ca-certificates -f
-RUN apt-get install -y openjdk-8-jre-headless
-RUN apt-get clean
-RUN rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+RUN apt-get install -y openjdk-8-jre-headless && \
```
Comment (nfultz) [inline Dockerfile]: you can probably float the update-ca-certs up a step, then combine the java steps into one layer.

File: Dockerfile [hunk 2]
```diff
@@ -1,7 +1,20 @@
-FROM boritzio/docker-base
-RUN add-apt-repository ppa:openjdk-r/ppa
-RUN apt-get update
+FROM phusion/baseimage
```
Comment (cwienberg) [inline Dockerfile]: You should pin this to a tag.

File: Dockerfile [hunk 3]
```diff
@@ -1,7 +1,20 @@
...
+ENV DEBIAN_FRONTEND=noninteractive
+#add this for mustache templates in config files
+ADD https://raw.githubusercontent.com/tests-always-included/mo/master/mo /usr/bin/
+RUN chmod +x /usr/bin/mo
```
Comment (cwienberg) [inline Dockerfile]: This only makes `mo` executable for the root user (something I've run into before). We should `a+x` this.

File: Dockerfile (hunk 1 above)
Comment (cwienberg) [reply Dockerfile]: Agreed.

**Comment by cwienberg** [comment]
I never understood the motivation for having the java stuff outside of `dev`. I think @nzewail recently tried working around this decision.

**Pull request merged**